packages/zipkin-transport-http/test/integrationTest.js

```diff
@@ -0,0 +1,40 @@
+/* eslint-disable no-console */
```

Comment:
I don't think this is needed? I cannot see any console.log statements in this file, have I missed one?